src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java

```diff
@@ -191,7 +197,21 @@ private static NettyChannelBuilder newNettyChannelBuilder(String targetUrl, Stri
    * @throws IOException in case the call credentials can't be constructed.
    */
   @Nullable
-  public static CallCredentials newCallCredentials(AuthAndTLSOptions options) throws IOException {
+  public static CallCredentials newCallCredentials(
+      Reporter reporter,
+      Map<String, String> clientEnv,
+      FileSystem fileSystem,
+      AuthAndTLSOptions options) throws IOException {
+    Credentials creds = newCredentials(reporter, clientEnv, fileSystem, options);
+    if (creds != null) {
+      return MoreCallCredentials.from(creds);
+    }
+    return null;
+  }
+
+  @Nullable
+  @VisibleForTesting
+  public static CallCredentials newCallCredentialsForTesting(AuthAndTLSOptions options) throws IOException {
```

Comment:
Nit: should this be called `newGoogleCallCredentialsForTesting`? This would makes the names in this file a bit more consistent (`Google` in the name means we only look at flags, otherwise we also look at the `.netrc`).